osxkeychain/osxkeychain_darwin_test.go

```diff
@@ -1,6 +1,8 @@
 package osxkeychain
 
 import (
+	"errors"
+	"fmt"
 	"github.com/docker/docker-credential-helpers/credentials"
 	"testing"
```

Comment:
hm, not sure what formatting is used in this repo, but ideally would be;

```go
import (
	"errors"
	"fmt"
	"testing"

	"github.com/docker/docker-credential-helpers/credentials"
)
```

Just a nit, so not a blocker